prow/cluster/build_serviceaccounts.yaml

```diff
@@ -0,0 +1,10 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  annotations:
+    iam.gke.io/gcp-service-account: k8s-infra-gcr-promoter@k8s-gcr-audit-test-prod.iam.gserviceaccount.com
```

Comment:
nit: Do we want people to comment or annotate these with their intended uses in order to discourage permission aggregation and misuse? If so we should start setting a precedent for that and encourage it in the docs.

Reply:
Done